mlflow/store/tracking/rest_store.py

```diff
@@ -891,18 +896,63 @@ def create_logged_model(
         Returns:
             The created model.
         """
+        # Include the first 100 params in the initial request
+        initial_params = []
+        remaining_params = []
+        if params:
+            initial_batch_size = MLFLOW_CREATE_LOGGED_MODEL_PARAMS_BATCH_SIZE.get()
+            initial_params = params[:initial_batch_size]
+            remaining_params = (
+                params[initial_batch_size:] if len(params) > initial_batch_size else []
+            )
```

Comment:
```suggestion
            remaining_params = params[initial_batch_size:]
```

this should also work.